### PR TITLE
RVSP System: First Pass

### PR DESCRIPTION
**Pull request by mxc-maggiechen**

### CODE REVIEW

File: frontend/data/components/rvsp.tsx
Comment (mxc-maggiechen) [inline frontend/data/components/rvsp.tsx:1]: barebones modal for rvsp

File: frontend/src/components/events/PostCard.tsx
Comment (mxc-maggiechen) [inline frontend/src/components/events/PostCard.tsx:1]: add rvsp to postcard components

File: frontend/src/lib/actions.ts
Comment (mxc-maggiechen) [inline frontend/src/lib/actions.ts:1]: handle form data

File: frontend/src/lib/nodemailer.ts
Comment (mxc-maggiechen) [inline frontend/src/lib/nodemailer.ts:1]: global mailer

File: frontend/package.json [hunk 1]
```diff
@@ -21,7 +23,10 @@
     "date-fns": "^3.6.0",
     "jose": "^5.6.3",
     "lucide-react": "^0.400.0",
+    "moment-timezone": "^0.5.45",
     "next": "14.2.3",
+    "node-cron": "^3.0.3",
+    "nodemailer": "^6.9.14",
```
Comment (xusandrew) [inline frontend/package.json:28]: can you uninstall pakages not being used anymore

File: frontend/src/components/events/PostCard.tsx [hunk 2]
```diff
@@ -25,6 +36,7 @@ export function Card({ event }: { event: Event }) {
           {event.start_time ? timeFromNow(event.start_time) : ''}
         </span>
       </div>
+      <button onClick={handleRVSPButton}>OPEN</button>
```
Comment (xusandrew) [inline frontend/src/components/events/PostCard.tsx]: are we styling this rn?

File: frontend/src/components/ui/InstaTest.tsx
Comment (xusandrew) [inline frontend/src/components/ui/InstaTest.tsx:1]: brother

**Review by xusandrew — CHANGES_REQUESTED** [review]

File: frontend/src/components/events/PostCard.tsx (hunk 2 above)
Comment (mxc-maggiechen) [reply frontend/src/components/events/PostCard.tsx]: i just added some simple styling

Screenshot 2024-07-08 052236

**Review by xusandrew — APPROVED** [review]

**Pull request merged**